### ISSUE TITLE
Getting information about the linear model

### ISSUE THREAD

**Issue by EoinKenny** [marcotcr/lime#201]
Hi, 

I've been trying to get more information about the linear model which LIME makes. Specifically, I would like to know the coefficients learned, not the contributions etc.

Is there a simple way to get a list of the weights learned by the local linear model?

Thanks in advance!

**Comment by marcotcr** [comment]
`exp.as_map()`

**Comment by EoinKenny** [comment]
Hi, thanks for the reply.

I see the thread is closed, but I just have two follow up questions if you have the time. I'm working on a binary classification problem here with a neural network with one output sigmoid unit.

1: exp.as_map() gives the contributions no? So I would have to divide by the input feature values to get the actual coefficients?
2: I ask because if you do add up those contributions and the intercept, you get the local probability/prediction. But wouldn't it have to be passed through a sigmoid function to get the probability? 

I suppose I'm confused about the pipeline here, presumable LIME uses a model like softmax or logistic regression for local probabilities and linear ridge regression for regression?

What are the actual weights for each feature which are multiplied by the input values, which are then passed to a (eg) logistic regression for the local approximation?

Hope that makes sense, sorry if I misunderstand the library.

**Comment by marcotcr** [comment]
No, `exp.as_map` actually gives you the coefficients. However, if you are using the default parameters, every feature is discretized, and thus coefficients and contributions become the same thing (every feature becomes binary, basically). I imagine this is where the confusion is coming from.
Let me know otherwise!

**Comment by EoinKenny** [comment]
Thanks for the reply!
